docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
+Kurzfristig wurde eine Team interne Evaluation der einzelnen Mitglieder durchgeführt.
+Das half einerseits dabei, dass Probleme gelöst und die Teammotivation sowie die Produktivität stieg, als auch das generelle Teamklima weiter gebessert wurde.
+\subsubsection{Product Increment}
+Das Produkt wurde um die Grundfunktionen erweitert.
+Es ist nun möglich, Tags anzulegen und entsprechend für diese Namen, Musik und ein Bild einzustellen.
+Genauso ist es dem Nutzer nun möglich, alle angelegten Tags zu durchsuchen.
+Es wurde weiterhin auch das Design Mockup von dem von Macio gestellten Designer in Vue größtenteils umgesetzt.
+Gleichzeitig wurden hierbei auch für eine leichtere Weiterarbeit jegliche Komponenten als möglichst allgemeine Vue Komponenten angelegt, um ein einheitliches Design zu ermöglichen.
+Für die Entwickler wurden auch die Pull Request Tests um Typescript Support erweitert.
+\subsubsection{Retrospektive}
+Besonders gut gefiel dem Team das Teamklima, vor allem des ehrliche Miteinander.
+Trotz Komplikationen und Probleme konnte am Ende dem Kunden ein schönes Product Increment geliefert werden.
+Pull Requests und Tickets wurden besser beschrieben, sodass Probleme besser verständlicher waren.
+Vor allem durch das interne Team Review wurde die Zusammenarbeit weiter gebessert.
+Das Team konnte weiterhin das Wissen mit Vue ausbauen, vor allem zu Reaktivität, Events und Kommunikation zwischen Komponenten.
+Stark bemängelt wurde der Scope des Meilensteins.
+Das Zeitmanagement war an dieser Stelle nicht gut, das Team hatte sich zu viel vorgenommen und am Ende zu wenig Zeit.
+Das Team stimmte hier überein, dass eine bessere Planung sowie Abstimmung für die nächsten Meilensteine notwendig wäre.
+Auch sollte eine frühere interne Deadline gesetzt werden, damit nicht kurz vor dem Sprintende noch viele Fehler auftreten und behoben werden müssen.
+Um insgesamt auch die Arbeit kontinuierlicher zu gestalten, sollten Aufgaben in kleinere Probleme aufgeteilt werden.
+
+\subsection{Meilenstein 5}
+Der Meilenstein fünf vom 14.01.2021 bis zum 28.01.2021 sollte dafür dienen, die Entwicklung am Produkt weitgehend zu beenden, sodass sich das Team darauf auf den Bericht fokussieren konnte.
+\subsubsection{Ziel}
+Geplant war, die letzten Anwendungszenarien final umzusetzen, sodass die letzten zwei Meilensteine höchstens zum Beheben kleinerer Fehler benötigt werden würden.
+Somit war das Ziel die Umsetzung der Einstellungen für die \textit{Leek Box} selbst als auch der einzelnen Tags.
+Die User Interface Elemente sollte, sofern noch nicht geschehen, ein einheitliches Design erhalten.
+Auch sollte die Dokumentation zur Bedienung der \textit{Leek Box} fertig gestellt sein und die Arbeit für den Bericht beginnen.
+\subsubsection{Probleme}
+Wie beim vorherigen Meilenstein hat auch hier das Zeitmanagement nicht komplett funktioniert.
+Gegen Ende fehlte es an Zeit, um neue umgesetzte Tickets auf Fehler zu testen und entsprechend fehlerfrei dem Kunden zu präsentieren.
```

Comment:
```suggestion
Gegen Ende fehlte es an Zeit, um neu umgesetzte Tickets auf Fehler zu testen und entsprechend fehlerfrei dem Kunden zu präsentieren.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee